python-avd/pyavd/_eos_designs/structured_config/network_services/utils_wan.py

```diff
@@ -71,6 +72,7 @@ def _filtered_wan_policies(self: AvdStructuredConfigNetworkServicesProtocol) ->
                 )
                 raise AristaAvdInvalidInputsError(msg)
 
+            # TODO: remove as_dict and fix the rest of utils_wan in another PR
```

Comment:
Please do add it in the list (https://github.com/aristanetworks/avd-internal/issues/205)

Reply:
that's a PR for me :) I will tackle it as part of wan_utils refactor (before 6.0)